alma/lib/Factories/LoggerFactory.php

```diff
@@ -22,56 +22,28 @@
  * @license   https://opensource.org/licenses/MIT The MIT License
  */
 
-namespace Alma\PrestaShop;
+namespace Alma\PrestaShop\Factories;
 
-use Alma\PrestaShop\Helpers\SettingsHelper;
-use Psr\Log\AbstractLogger;
-use Psr\Log\LogLevel;
+use Alma\PrestaShop\Model\LoggerPsr1;
+use Alma\PrestaShop\Model\LoggerPsr3;
 
 if (!defined('_PS_VERSION_')) {
     exit;
 }
 
-class Logger extends AbstractLogger
+class LoggerFactory
 {
     public static function instance()
     {
         static $instance;
-
         if (!$instance) {
-            $instance = new Logger();
+            if (version_compare(_PS_VERSION_, '9', '>=')) {
+                $instance = new LoggerPsr3();
+            } else {
+                $instance = new LoggerPsr1();
+            }
```

Comment:
To avoid `else`, we can edit with this code:
```

$instance = new LoggerPsr3();
if (version_compare(_PS_VERSION_, '9', '<')) {
    $instance = new LoggerPsr1();
}
```
WDYT ?